spring-boot-jib/pom.xml

```diff
@@ -0,0 +1,68 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>spring-boot-jib</artifactId>
+    <version>0.1-SNAPSHOT</version>
+    <name>spring-boot-jib</name>
+    <packaging>war</packaging>
```

Comment:
Why are we producing a war? Could we do a jar instead?

Reply:
Yup, will modify it

Reply:
Removed the packaging